include/nuttx/sensors/sensor.h

```diff
@@ -1031,6 +1031,10 @@ struct sensor_lowerhalf_s
 
   int type;
 
+  /* Device number */
+
+  int devno;
```

Comment:
but nobody use it?

Reply:
I'll add sensors using this field in next PRs

Reply:
since the common code doesn't use it, should we move to individual driver who need it?

Reply:
yes, you are right. It's better to keep it in private sensor data. I'm closing this PR